zwave_js_server/model/driver.py

```diff
@@ -65,6 +70,14 @@ async def async_get_log_config(self) -> LogConfig:
         result = await self._async_send_command("get_log_config", require_schema=4)
         return LogConfig.from_dict(result["config"])
 
+    async def async_start_listening_logs(self) -> None:
```

Comment:
Side note: We've talked about removing the `async_` prefix from all names, since everything should run in async context. Something for the future.